Deeploy/Targets/Generic/Parsers.py

```
@@ -2747,3 +2806,64 @@ def parseNodeCtxt(self,
                 "ch_im_out"] * self.operatorRepresentation["dim_im_out_y"]
             return newCtxt, True
         return ctxt, False
+
+
+############################
+
+
+class PowParser(NodeParser):
```

Ruff (0.14.5)

Redefinition of unused `PowParser` from line 1967

(F811)